templates/rateUpdateTemplate.php

```diff
@@ -0,0 +1,49 @@
+<?php include 'inc/header.php'; ?>
+<form method="post" action="rateUpdate.php">
+<div class="form-group">
+        <label>Choose Rate Type</label>
+        <select name="rateTypeChange" class="form-control">
+            <option value="0"> </option>
+            <option value="Charging">Charging</option>
```

Comment:
we should do a foreach loop here over the ratetype table instead of hardcoding it

Reply:
fixed!